src/poliastro/plotting/orbit/backends/matplotlib_backend.py

```diff
@@ -0,0 +1,282 @@
+"""A module implementing orbit plotter backends based on Matplotlib."""
```

Comment:
Maybe rename the module from `backends/matplotlib_backend.py` to `backends/matplotlib.py`?